packages/@aws-cdk/aws-ecr/README.md

```diff
@@ -79,6 +79,26 @@ You can set tag immutability on images in our repository using the `imageTagMuta
 new ecr.Repository(this, 'Repo', { imageTagMutability: ecr.TagMutability.IMMUTABLE });
 ```
 
+### Encryption
+
+By default a Repository is server-side encrypted with AES-256 by AWS. However, you can use AWS managed KMS or customer managed KMS. When `encryptionKey` is set, the `encryption` must be `KMS` or empty.
+
+```ts
+new ecr.Repository(this, 'Repo', {
+  encryption: ecr.RepositoryEncryption.KMS
```

Comment:
Can you include a comment in the example that explains this will use a AWS managed KMS key.